src/library/form/LibraryForm.jsx

```diff
@@ -0,0 +1,251 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import { Button, Icon, Input, StatefulButton, StatusAlert, ValidationFormGroup } from '@edx/paragon';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import { LIBRARY_TYPES, SUBMISSION_STATUS, libraryShape } from '../common';
+import { createLibrary, libraryFormInitialState, resetForm, selectLibraryForm } from './data';
+
+import messages from './messages';
+
+class LibraryForm extends React.Component {
```

Comment:
Would it be fair to be a bit more explicit and call this something like `LibraryCreateForm` or `CreateLibrayForm`?

Reply:
Done!